applet/src/keyb.c

```diff
@@ -119,20 +128,17 @@ void handle_hotkey( int keycode )
             syslog_dump_dmesg();
             break ;
         case 7 :
```

Comment:
I asked you to give me some feedback about the keys in https://groups.google.com/forum/#!topic/md380tools/NYdVOy0NI0c
You asked me just to do a PR ... may be I misunderstood this, feel free to let me feel like a fool.